app/src/main/jni/transition/PeerManager.c

```diff
@@ -362,7 +363,7 @@ Java_com_breadwallet_wallet_BRPeerManager_putPeer(JNIEnv *env, jobject thiz, jby
 
 JNIEXPORT void JNICALL
 Java_com_breadwallet_wallet_BRPeerManager_createPeerArrayWithCount(JNIEnv *env, jobject thiz,
-                                                                   size_t peerCount) {
+                                                                   jint peerCount) {
```

Comment:
Hey @vsima this is cruft....if I changed, it I made a mistake. I wasn't trying to touch peerCount at all